tests/regression/facebook/test_chrome_facebook_ail_click_close_chat_tab.sikuli/test_chrome_facebook_ail_click_close_chat_tab.py

```diff
@@ -0,0 +1,47 @@
+# if you are putting your test script folders under {git project folder}/tests/, it will work fine.
+# otherwise, you either add it to system path before you run or hard coded it in here.
+sys.path.append(sys.argv[2])
+import os
+import common
+import facebook
+import shutil
+import browser
+import time
+
+# Disable Sikuli action and info log
+com = common.General()
+com.infolog_enable(0)
+
+chrome = browser.Chrome()
+fb = facebook.facebook()
+
+chrome.clickBar()
+chrome.enterLink(sys.argv[3])
+fb.wait_for_loaded()
+
+sleep(2)
+setAutoWaitTimeout(10)
```

Comment:
try to remove all the `setAutoWaitTimeout(10)`.
Please see #639 for more information! Thanks.